app/components/home-page/organization-dashboard/index.hbs

```diff
@@ -20,6 +22,11 @@
 
   {{! #id is required for scroll-to-top util }}
   <main id='ak-organization-dashboard-main' local-class='dashboard-main'>
+    <HomePage::OrganizationDashboard::OnboardingGuide
+      @showOnboardingGuide={{this.showOnboardingGuide}}
+      @onToggleOnboardingGuide={{this.onToggleOnboardingGuide}}
+    />
```

Comment:
wrap the component in if condition instead of passing `showOnboardingGuide` and move the component outside of main container